connectors/kafka/src/main/java/forklift/producers/KafkaForkliftProducer.java

```diff
@@ -115,7 +115,17 @@ public KafkaForkliftProducer(String topic, KafkaProducer<?, ?> kafkaProducer) {
 
     @Override
     public String send(String message) throws ProducerException {
-        return sendForkliftWrappedMessage(message, null);
+        // let the schema registry just treat it as an avro string and ignore producer properties
+        ProducerRecord record = new ProducerRecord<>(topic, null, message);
```

Comment:
Well, I wanted to write just raw role message strings to the role-input source, assuming properties weren't relevant. This may not be the right thing to do, but there are a few things to consider here:
- The schema registry will not allow us to produce fundamentally different schemas to the same topic (i.e. the `string` schema and the forklift wrapping schema in this case, so our conditional wrapping of messages is probably wrong), because they aren't compatible.
- Also, I don't really see a reason to have Forklift GUI and Forklift produce different formats for role messages.
- Producer properties feel like bloat to me, because they complicate the responsibility of `ForkliftProducerI`. And really, "producer properties" aren't even _hard_ to do on the client side.

This last bit is the key here: in my opinion, `ForkliftProducerI` has too many responsibilities:
 - It's supposed to send data.
 - It's also supposed to serialize data.
 - It's also supposed to make sure certain information is carried with the data, whether it's already serialized or not.

The last responsibility kind of made sense when AMQ was handling that for us, but it really doesn't make sense as a core part of a producer that uses Kafka.

Because of this, `ForkliftProducerI` can't be used for _just_ sending data without the other two responsibilities becoming part of the problem. Then to get what I want out of `ForkliftProducerI` (just sending), I have to dip into its responsibilities so that they can be duplicated on the GUI side.

One of a few things has to happen here:
- Forklift GUI needs to know how to perform `ForkliftProducerI`'s responsibilities (knowing how to wrap also requires knowing how to serialize) - seems too distributed.
- `ForkliftProducerI` is redesigned to have it's responsibilities be more amenable. Also seems like a lot of work.
- Something in Forklift, outside of `ForkliftProducerI`, needs to know or expose how `ForkliftProducerI` performs this wrapping so that it can be done on behalf of Forklift GUI (either through duplication, or though a more careful redesign). This seems like a lot of work, when I don't really _need_ any wrapping. It's annoying to complicate the interfaces. Add a `sendRaw` to the producer interface? Then what about AMQ which has no problem with doing that with send? Expose a "raw producer" in the connector interface? Seems ugly and already halfway to the previous solution.
- Ignore a bloated responsibility in one part of one implementation of`ForkliftProducerI` and risk breaking clients that expect it to perform all of its responsibilities all the time.